src/main/java/lotto/LottoMain.java

```diff
@@ -9,26 +9,31 @@
 import java.util.Arrays;
 import java.util.List;
 
-import static lotto.util.Util.floorDiv;
-import static lotto.util.Util.split;
+import static lotto.util.ListUtil.parseIntegerList;
+import static lotto.util.StringUtil.split;
 import static lotto.view.InputView.*;
 import static lotto.view.OutView.printLottoList;
 import static lotto.view.OutView.printResult;
 
 public class LottoMain {
     public static void main(String[] args) {
-        UserInput userInput = new UserInput(inputMoney());
+        UserInput userInput = new UserInput(inputMoney(), inputPassiveNumber());
 
         LottoService lottoService = new LottoService(generateLottoList(userInput), new MatchPointRank());
-        printLottoList(lottoService.getLottoList());
+        printLottoList(lottoService.getLottoList(), userInput);
 
         lottoService.playLotto(Arrays.asList(split(inputWinningNumbers())), inputBonusNumber());
-        printResult(userInput.getMoney(), lottoService.getMatchPointMap());
+        printResult(userInput, lottoService.getMatchPointMap());
     }
 
     private static List<Lotto> generateLottoList(UserInput userInput) {
         List<Lotto> lottoList = new ArrayList<>();
-        for (int i = 0; i < userInput.floorDivMoney(); i++) {
+        for(int i = 0; i < userInput.getMassiveNumber(); i++) {
+            List<String> numbers = Arrays.asList(split(inputPassiveNumbers()));
+            Lotto lotto = new Lotto(parseIntegerList(numbers));
+            lottoList.add(lotto);
+        }
+        for (int i = 0; i < userInput.getFloorDivCount() - userInput.getMassiveNumber(); i++) {
```

Comment:
```suggestion
        for (int i = 0; i < userInput.autoCount(); i++) {
```

값을 꺼내와서 상태를 건드리는것 보다,
상태를 가진 객체에게 물어보는건 어떨까요?